app/services/npq/accept.rb

```diff
@@ -20,6 +20,8 @@ def call
         create_profile
         npq_application.update(lead_provider_approval_status: "accepted") && other_applications.update(lead_provider_approval_status: "rejected")
       end
+    rescue ActiveRecord::StatementInvalid
```

Comment:
are you not able to check if it's been accepted or not rather than rely on `ActiveRecord::StatementInvalid` to infer that it maybe due to the application having already been accepted. as `ActiveRecord::StatementInvalid` could be raised due to another issue